src/main/java/edu/kis/powp/jobs2d/features/CommandParser.java

```diff
@@ -0,0 +1,97 @@
+package edu.kis.powp.jobs2d.features;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.drivers.adapter.JsonAdapter;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class CommandParser {
```

Comment:
1. Please provide an interface for CommandParser to separate txt and json implementations and to enable implementing support for different formats in the future. 
2. I do not see the reason why the parser should restrict input only to files. Please use string as an input and make parser independent from java.io.*

Reply:
ad 1 Interface provided. Separate implementations for txt and json files implemented

ad 2 File argument changed for String for parser implementations